README.md

```diff
@@ -509,7 +491,7 @@ Previous releases may also have been signed with one of the following GPG keys:
 
 ### Working Groups
 
-Information on the current Node.js Working Groups can be found in the
+Information on the current Ayo.js Working Groups can be found in the
```

Comment:
I realise that this PR is currently a find and replace, but I think my above comment is worth doing now.